specification/applicationinsights/ApplicationInsights.LiveMetrics/models.tsp

```diff
@@ -0,0 +1,567 @@
+ */
+#suppress "@azure-tools/typespec-azure-core/casing-style" "Need to follow the casing style of the existing SDK contract."
+model CollectionConfigurationInfo {
+  @doc("An encoded string that indicates whether the collection configuration is changed.")
+  ETag: string;
+
+  @doc("An array of metric configuration info.")
+  Metrics: DerivedMetricInfo[];
+
+  @doc("An array of document stream configuration info.")
+  DocumentStreams: DocumentStreamInfo[];
+
+  @doc("Controls document quotas to be sent to QuickPulse.")
+  QuotaInfo?: QuotaConfigurationInfo;
+}
+
+/**
+ * Controls document quotas to be sent to QuickPulse.
+ */
+#suppress "@azure-tools/typespec-azure-core/casing-style" "Need to follow the casing style of the existing SDK contract."
+model QuotaConfigurationInfo {
+  @doc("Initial quota")
+  InitialQuota?: float32;
+
+  @doc("Max quota")
+  MaxQuota: float32;
+
+  @doc("Quota accrual rate per second")
+  QuotaAccrualRatePerSec: float32;
+}
+
+/**
+ * Configurations/filters set by UX to scope the document/telemetry it's interested in.
+ */
+#suppress "@azure-tools/typespec-azure-core/casing-style" "Need to follow the casing style of the existing SDK contract."
+model DocumentStreamInfo {
+  @doc("Identifier of the document stream initiated by a UX.")
+  Id: string;
+
+  @doc("Gets or sets an OR-connected collection of filter groups.")
+  DocumentFilterGroups: DocumentFilterConjunctionGroupInfo[];
+}
+
+/**
+ * A collection of filters for a specific telemetry type.
+ */
+#suppress "@azure-tools/typespec-azure-core/casing-style" "Need to follow the casing style of the existing SDK contract."
+model DocumentFilterConjunctionGroupInfo {
+  @doc("Telemetry type.")
+  TelemetryType:
+    | "Request"
+    | "Dependency"
+    | "Exception"
+    | "Event"
+    | "Metric"
+    | "PerformanceCounter"
+    | "Trace";
+
+  @doc("An array of filter groups.")
+  Filters: FilterConjunctionGroupInfo;
+}
+
+/**
+ * A metric configuration set by UX to scope the metrics it's interested in.
+ */
+#suppress "@azure-tools/typespec-azure-core/casing-style" "Need to follow the casing style of the existing SDK contract."
+model DerivedMetricInfo {
+  @doc("metric configuration identifier.")
+  Id: string;
+
+  @doc("Telemetry type.")
+  TelemetryType: string;
+
+  @doc("A collection of filters to scope metrics that UX needs.")
+  FilterGroups: FilterConjunctionGroupInfo[];
+
+  @doc("Telemetry's metric dimension whose value is to be aggregated. Example values: Duration, Count(),...")
+  Projection: string;
+
+  @doc("Aggregation type. This is the aggregation done from everything within a single server.")
+  Aggregation: "Avg" | "Sum" | "Min" | "Max";
+
+  @doc("Aggregation type. This Aggregation is done across the values for all the servers taken together.")
+  BackEndAggregation: "Avg" | "Sum" | "Min" | "Max";
```

Comment:
This should be 1 or 2 enums depending on whether the values will always be the same or not.